vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/SequenceOverviewManager.java

```diff
@@ -0,0 +1,48 @@
+package edu.asu.diging.vspace.core.services.impl;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+
+import edu.asu.diging.vspace.core.data.ModuleRepository;
+import edu.asu.diging.vspace.core.data.SequenceRepository;
+import edu.asu.diging.vspace.core.data.display.ModuleLinkDisplayRepository;
+import edu.asu.diging.vspace.core.model.ModuleOverview;
+import edu.asu.diging.vspace.core.model.SequenceOverview;
+import edu.asu.diging.vspace.core.model.impl.Sequence;
+
+@Component
+public class SequenceOverviewManager {
```

Comment:
this class looks like it should be merged with `SequenceOverviewJsonFormat`. It doesn't seem to do anything special besides getting the sequences for a module.

Reply:
Merged sequenceoverviewjsonformat into this class